hack/gotest.sh

```diff
@@ -35,7 +35,12 @@ else
     JQ_FILTER='select(has("Output") and (.Action=="output") and (has("Test")|not) and (.Output!="PASS\n") and (.Output!="FAIL\n") and (.Output|startswith("coverage:")|not) and (.Output|contains("[no test files]")|not)) | .Output'
 fi
 
-echo "go test $@"
+if [ "$#" -le 6 ]; then
```

Comment:
I don't understand the special role of 6 or the logic behind it - it feels like we should rely on something explicit instead?